x-pack/plugin/wildcard/src/main/resources/org/elasticsearch/xpack/wildcard/org.elasticsearch.xpack.wildcard.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0 and the Server Side Public License, v 1; you may not use this file except
+# in compliance with, at your election, the Elastic License 2.0 or the Server
+# Side Public License, v 1.
+#
+
+class org.elasticsearch.xpack.wildcard.WildcardDocValuesField @dynamic_type {
+  String get(String)
```

Comment:
I don't think the methods need to be whitelisted, just note that this is a subclass of `AbstractKeywordDocValuesField`.

Reply:
Done.